### PR TITLE
docs: README Quick Start section tweaked

### PR DESCRIPTION
**Pull request by kik-kik**
# README Quick Start section tweaked

## Description
Based on my experience trying to set up the project on M1 mac I decided to make minor tweaks to the `Quick Start` section to emphasize the step for running terminal with Rosetta prior to doing anything else to avoid the next person from running into the same issues as myself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
----
-
-Ensure Python 3.8+ is available on your machine (see [this guide](https://docs.python-guide.org/starting/install3/osx/) for instructions if you're on a mac and haven't installed anything other than the default system Python.)
 
-For some functionality Java JDK 11+ is also required (e.g. [AdoptOpenJDK](https://adoptium.net/?variant=openjdk11)) with `$JAVA_HOME` set, and maven is needed for downloading jar dependencies.
-Maven is available from your package manager in most Linux distributions and from [homebrew](https://brew.sh/) on mac, or you can install yourself by [downloading a binary](https://maven.apache.org/download.cgi) and following maven's [install instructions](https://maven.apache.org/install.html).
-The [Rosetta 2 terminal](https://support.apple.com/en-ca/HT211861) is also suggested for installing the above requirements on Apple Silicon (M1).
+## Quick Start
+> Apple Silicon (M1) user suggestion
+>
```
Comment (wlach) [inline README.md:14]:
```suggestion
## Quick Start

> Apple Silicon (M1) user suggestion
>
```

File: README.md [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
 
-* (For Mozilla Employees or Contributors not in Data Engineering) Set up GCP command line tools, [as described on docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org/cookbooks/bigquery/access.html#using-the-bq-command-line-tool). Note that some functionality (e.g. writing UDFs or backfilling queries) may not be allowed.
-* (For Data Engineering) In addition to setting up the command line tools, you will want to log in to `shared-prod` if making changes to production systems. Run `gcloud auth login --update-adc --project=moz-fx-data-shared-prod` (if you have not run it previously).
+### GCP CLI tools
+- **For Mozilla Employees or Contributors (not in Data Engineering)** - Set up GCP command line tools, [as described on docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org/cookbooks/bigquery/access.html#using-the-bq-command-line-tool). Note that some functionality (e.g. writing UDFs or backfilling queries) may not be allowed.
```
Comment (wlach) [inline README.md:24]:
```suggestion
### GCP CLI tools

- **For Mozilla Employees or Contributors (not in Data Engineering)** - Set up GCP command line tools, [as described on docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org/cookbooks/bigquery/access.html#using-the-bq-command-line-tool). Note that some functionality (e.g. writing UDFs or backfilling queries) may not be allowed.
```

File: README.md [hunk 3]
```diff
@@ -1,67 +1,69 @@
...
+- **For Data Engineering** - In addition to setting up the command line tools, you will want to log in to `shared-prod` if making changes to production systems. Run `gcloud auth login --update-adc --project=moz-fx-data-shared-prod` (if you have not run it previously).
 
-Clone the repository
+### Installing bqetl library
```
Comment (wlach) [inline README.md:27]:
```suggestion
### Installing bqetl library

```

**Review by wlach — APPROVED** [review]
Looks good, just some minor formatting suggestions

**Comment by kik-kik** [comment]
@wlach thanks for the feedback, I committed your suggestions. 🙇

**Pull request merged**